src/components/UserInfo.js

```diff
@@ -1,22 +1,29 @@
 export default class UserInfo {
-  constructor({ profileName, profileJob }) {
+  constructor({ profileName, profileJob, profileAvatar, isLiked }) {
     this._profileName = document.querySelector(profileName);
     this._profileJob = document.querySelector(profileJob);
+    this._profileAvatar = document.querySelector(profileAvatar);
+    this._isLiked = document.querySelector(isLiked);
   }
 
   getUserInfo() {
     //returns user information
     return {
       profileName: this._profileName.textContent,
       profileJob: this._profileJob.textContent,
+      profileAvatar: this._profileAvatar.src,
     };
   }
 
-  setUserInfo({ profileName, profileJob }) {
+  setUserInfo({ profileName, profileJob, profileAvatar }) {
     this._profileName.textContent = profileName;
     this._profileJob.textContent = profileJob;
 
```

Comment:
Please, set the avatar inside `setUserInfo`, otherwise the default avatar is displayed when the project starts

```js

  setUserInfo({ profileName, profileJob, profileAvatar }) {
    this._profileName.textContent = profileName;
    this._profileJob.textContent = profileJob;
    this.setUserAvatar({ profileAvatar }); // here
  }

```